src/App.js

```diff
@@ -1,8 +1,37 @@
 import './App.css';
+import Tasks from './todo.json'
+import { useState } from "react"
 
 function App() {
-  return (
-    <></>
+
+  const [indexCheck, setIndexCheck] = useState([])
+
+  function change(event, index) {
+    const { checked } = event.target
+  if(checked===true){
```

Comment:
@PuniaMohit why checking checked === true? is if(checked) not working?

Reply:
sir by this condition, I am checking index number of that tasks is in array or not. I made an array which consits index number of all selected tasks.